### PR TITLE
whsniff: init

### PR DESCRIPTION
**Pull request by snicket2100**
###### Motivation for this change

adding a new package

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by snicket2100** [comment]
labelled it as `platforms.linux` as I have no way to build it on Darwin... is this all right, or what's the alternative?

**Review by calbrecht — APPROVED** [review]
Build with `nixpkgs-review pr` and started whsniff in resulting shell. 
LGTM, although i cannot tell if it does what it should, since i do not own a TI CC2531 USB Dongle :)

**Comment by snicket2100** [comment]
:) I did test the binary myself with the dongle, works as expected!

**Comment by calbrecht** [comment]
`nixpkgs-review pr 85573`
1 package built:
whsniff

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7102,6 +7102,12 @@
     githubId = 602439;
     name = "Serguei Narojnyi";
   };
+  snicket2100 = {
+    email = "57048005+snicket2100@users.noreply.github.com";
+    github = "snicket2100";
+    githubId = 57048005;
+    name = "snicket2100";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix:7110]: Please make this into a separate commit:

````
maintainers: add <name>
```

Comment (snicket2100) [reply maintainers/maintainer-list.nix:7110]: @jonringer fixed

**Review by jonringer — COMMENTED** [review]
the `whsniff` commit should be `whisniff: init at 1.3`

**Comment by snicket2100** [comment]
> the `whsniff` commit should be `whisniff: init at 1.3`

fixed! @jonringer

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 85573 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  - whsniff</details>
